config/cluster-reader-rbac.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: machine-api-operator:cluster-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-cluster-reader: "true"
+rules:
+ apiGroups:
+  - machine.openshift.io
+  resources:
+  - machineconfigpools
+  - machineconfigs
```

Comment:
please remove machineconfigs and machineconfigpools, those should be handle by the machine config operator https://github.com/openshift/machine-config-operator